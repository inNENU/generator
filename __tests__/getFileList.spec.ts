import { expect, it } from "vitest";

import { getFileList } from "../src/index.js";

it("getFileList", () => {
  expect(getFileList("./src")).toMatchSnapshot();
});
