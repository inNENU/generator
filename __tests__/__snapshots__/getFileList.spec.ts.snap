// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`getFileList 1`] = `
[
  "ai.ts",
  "config.ts",
  "index.ts",
  "markdown.ts",
  "page.ts",
  "text.ts",
  "typings.ts",
  "utils.ts",
  "components/common.ts",
  "components/page.ts",
  "components/typings.ts",
  "components/account/json.ts",
  "components/account/markdown.ts",
  "components/account/typings.ts",
  "components/action/json.ts",
  "components/action/markdown.ts",
  "components/action/typings.ts",
  "components/audio/json.ts",
  "components/audio/markdown.ts",
  "components/audio/typings.ts",
  "components/card/json.ts",
  "components/card/markdown.ts",
  "components/card/typings.ts",
  "components/carousel/json.ts",
  "components/carousel/typings.ts",
  "components/doc/json.ts",
  "components/doc/markdown.ts",
  "components/doc/typings.ts",
  "components/doc/utils.ts",
  "components/footer/json.ts",
  "components/footer/typings.ts",
  "components/grid/json.ts",
  "components/grid/markdown.ts",
  "components/grid/typings.ts",
  "components/img/json.ts",
  "components/img/markdown.ts",
  "components/img/typings.ts",
  "components/list/json.ts",
  "components/list/markdown.ts",
  "components/list/typings.ts",
  "components/location/json.ts",
  "components/location/markdown.ts",
  "components/location/typings.ts",
  "components/phone/json.ts",
  "components/phone/markdown.ts",
  "components/phone/typings.ts",
  "components/table/json.ts",
  "components/table/markdown.ts",
  "components/table/typings.ts",
  "components/text/json.ts",
  "components/text/markdown.ts",
  "components/text/typings.ts",
  "components/title/json.ts",
  "components/title/markdown.ts",
  "components/title/typings.ts",
  "components/video/json.ts",
  "components/video/markdown.ts",
  "components/video/typings.ts",
  "generator/account.ts",
  "generator/index.ts",
  "generator/map.ts",
  "generator/music.ts",
  "generator/svgIcons.ts",
  "generator/typings.ts",
  "helpers/accessToken.ts",
  "helpers/checkAssets.ts",
  "helpers/getFileList.ts",
  "helpers/getFileMap.ts",
  "helpers/git.ts",
  "helpers/index.ts",
  "helpers/json.ts",
  "helpers/oss.ts",
  "helpers/promiseQueue.ts",
  "helpers/svg.ts",
  "helpers/wordCount.ts",
  "helpers/yaml.ts",
]
`;
